purview/api/v1.py

```diff
@@ -17,3 +18,13 @@ def about():
     :return: A JSON object with version info
     """
     return jsonify({'version': version})
+
+
+@api_v1.route('/<resource>/<id>')
+def get_resource(resource, id):
+    for model in all_models:
+        if model.__name__.lower() == resource:
+            try:
+                return model.nodes.get(id_=id).serialized
```

Comment:
You need to actually find the `UniqueIdProperty` to filter against since it's different names on different nodes (e.g. `username`).

You can find the uid with:
```python
for prop_name, prop_def in model.defined_properties().items():
     if isinstance(prop_def, UniqueIdProperty):
         print('{0} is the uid!'.format(prop_name))
```

Reply:
The serialized output should be wrapped in a `jsonify` call.

Reply:
Oh, you're right. Sorry about that!